tests/atoms/condlink.scm

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
I suggest adding one test for EvaluationLink with grounded predicate

Reply:
Hi @noskill my scheme knowledge is very limited, will this suffice
<pre>
(define (grounded_cond1 ) (cog-new-stv 0 0))

(define (grounded_cond2 ) (cog-new-stv 1 1))

(define grounded-cond
    (CondLink
	(Evaluation
	     (GroundedPredicate "scm:grounded_cond1")
	     (List ))
	(NumberNode 1)

	(Evaluation
	     (GroundedPredicate "scm:grounded_cond2")
	     (List ))
	(NumberNode 2)))
</pre>

Reply:
looks like it was added

Reply:
Yes, looks good, thanks